CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
```

Comment:
It’s gonna be 0.1.2. Could you please add `@since` annotations?

Reply:
Yep, done.